pkg/chrootarchive/archive.go

```diff
@@ -35,13 +48,17 @@ func Untar(tarArchive io.Reader, dest string, options *archive.TarOptions) error
 		}
 	}
 
-	decompressedArchive, err := archive.DecompressStream(tarArchive)
-	if err != nil {
-		return err
+	r := ioutil.NopCloser(tarArchive)
```

Comment:
The only disadvantage I see to doing this is if tarArchive is indeed a file, this would cause the later `cmd.Cmd` to use a pipe instead of directly using the file. So I must ask, does this really need to be wrapped in a closer if it will never be closed.